### DISCUSSION TITLE
Deployment

### DISCUSSION THREAD

**Discussion by maphew** [pcmacdon/pdq#7] [General]
After studying the Readme and [Setup doc](https://jsish.org/fossil/pdq/doc/tip/md/Setup.md) several more times I think I see the expected typical deployment as: 

**Machine A**: a desktop, laptop or whatever. It is used to author, administrate and otherwise create and edit the site and components. This computer runs Jsish and Fossil. Fossil is used for file versioning, synchronizing with the server, and local webserver testing. There can be several machines in this "A" role.

**Machine B**: a server, it runs Fossil as webserver. Jsish is not used. 

It is possible to combine both these roles onto a single machine, and that's typically happening for the developer but conceptually this dual layout is still in play. Meaning: I don't have to figure out get Jsish working on Dreamhost or NearlyFreeSpeech.net in order to use PDQ. As long as I have a Jsish in a linux vm somehwere I can still use it to control and author a website, while the site itself can serve users as long as fossil itself can run on that host regardless of whether Jsish can. 

Is this a good understanding or just my wishful thinking? ;-)

**Comment by pcmacdon** [comment]
Essentially yes, as long as you don't use or load a plugin that has backend logic. However the server code will not be there to support user login, for example, because backend code requires Jsish.
